drivers/spi/Kconfig.esp32

```diff
@@ -0,0 +1,34 @@
+# ESP32 SPI configuration
+
+# Copyright (c) 2020 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ESP32_SPIM
+	bool "ESP32 SPI Master driver"
+	depends on SOC_ESP32
+	default y
+	help
+	  Enables support for ESP32 SPI Master driver.
+
+if ESP32_SPIM
+
+config SPI_ESP32_HALFDUPLEX
+	bool "Half-duplex mode"
+	default n
```

Comment:
I think the "default n" lines in this file should be removed, see https://docs.zephyrproject.org/latest/guides/kconfig/tips.html#redundant-defaults